src/codegen/type/varchar_type.cpp

```diff
@@ -231,14 +322,24 @@ static std::vector<TypeSystem::ComparisonInfo> kComparisonTable = {
 // Unary operators
 static Ascii kAscii;
 static Length kLength;
+static Trim kTrim;
 static std::vector<TypeSystem::UnaryOpInfo> kUnaryOperatorTable = {
-    {OperatorId::Ascii, kAscii}, {OperatorId::Length, kLength}};
+    {OperatorId::Ascii, kAscii},
+    {OperatorId::Length, kLength},
+    {OperatorId::Trim, kTrim}};
 
 // Binary operations
 static Like kLike;
 static DateTrunc kDateTrunc;
+static BTrim kBTrim;
+static LTrim kLTrim;
+static RTrim kRTrim;
 static std::vector<TypeSystem::BinaryOpInfo> kBinaryOperatorTable = {
-    {OperatorId::Like, kLike}, {OperatorId::DateTrunc, kDateTrunc}};
+    {OperatorId::Like, kLike},
+    {OperatorId::DateTrunc, kDateTrunc},
+    {OperatorId::BTrim, kBTrim},
+    {OperatorId::LTrim, kLTrim},
+    {OperatorId::RTrim, kRTrim}};
```

Comment:
Run `clang-format`

Reply:
@pmenon This was the output after running the formatter.py script, any suggestions where to change?

Reply:
Ah, no change needed then. Wasn't sure if you'd run it already.